app/models/population.rb

```diff
@@ -1,4 +1,4 @@
-class Population < ApplicationRecord
+class Population < VersionedModel
```

Comment:
Are we sure we want to add this to versioned models? I'm not sure how important it is to keep a history of changes for the dashboard and that table is already pretty busy?

Reply:
Hmm... good question. Maybe we don't care about this actually 🤔

Reply:
I've removed now - good call raising this 👍